debugger/src/debugger.rs

```diff
@@ -403,6 +408,76 @@ impl DebugSession {
     }
 }
 
+/// Manage the active RTT target for a specific DebugSession, as well as provide methods to reliably move RTT from target, through the debug_adapter, to the client.
+pub(crate) struct DebuggerRttTarget {
```

Comment:
I'd change this name to something like `RttConnection` the name now is very confusing imo.

Reply:
Done. I was struggling on the creative side, so yeah, it was getting confusing :)